pantab/tests/test_types.py

```diff
@@ -0,0 +1,21 @@
+import tableauhyperapi as tab_api
+
+import pantab._types
+
+
+def test_read_varchar_type():
```

Comment:
Can you just parametrize this test instead?

```python
@pytest.mark.parametrize("nullable", [tab_api.Nullability.NULLABLE, tab_api.Nullability.NOT_NULLABLE])
def test_read_varchar_type(nullable):
   ...
```

should do the trick

Reply:
Changed the tet to be parameterized.